flaskProject/resources/motos.py

```diff
@@ -0,0 +1,77 @@
+from flask_restful import Resource, reqparse
+from models.moto_model import MotoModel
+
+
+class Moto(Resource):
+    def get(self, id):
+        try:
+            moto = MotoModel.find_by_id(id)
+            return {'moto':moto.json()}, 200
+        except:
+            return {"message": "Error Get Moto"}, 500
+
+    def post(self):
+        parser = reqparse.RequestParser()
+        parser.add_argument('state', type=str, required=True, help="This field cannot be left blank")
+        parser.add_argument('matricula', type=str, required=True, help="This field cannot be left blank")
+        parser.add_argument('date_estreno', type=str, required=True, help="This field cannot be left blank")
+        parser.add_argument('model_generic', type=str, required=True, help="This field cannot be left blank")
+        parser.add_argument('last_coordinate_latitude', type=float, required=True, help="This field cannot be left blank")
+        parser.add_argument('last_coordinate_longitude', type=float, required=True, help="This field cannot be left blank")
+        parser.add_argument('km_restantes', type=float, required=True, help="This field cannot be left blank")
+        parser.add_argument('km_totales', type=float, required=True, help="This field cannot be left blank")
+        parser.add_argument('date_last_check', type=str, required=True, help="This field cannot be left blank")
+        data = parser.parse_args()
+
+        try:
+            moto = MotoModel(data['state'], data['matricula'], data['date_estreno'], data['model_generic'], data['last_coordinate_latitude'],
+                               data['last_coordinate_longitude'], data['km_restantes'], data['km_totales'],
+                              data['date_last_check'])
+            MotoModel.save_to_db(moto)
+            return {"message": "Moto added successfully"}, 200
+        except:
+            return {"message": "Error Post Moto"}, 500
```

Comment:
El mensaje debería ser más aclaratorio para el usuario. Como: "Error adding the moto in the database"